docs/usage/configuring-plugins-in-stackfiles.rst

```diff
@@ -6,9 +6,9 @@ stackfiles. This section will show you how you can find out which configuration
 is possible for each plugin.
 
 Try the following command to see which plugins are available based on your
-conanfile profile, such as the example ``tests/profile_default.py``::
+conanfile profile, such as the example ``tests/conanfile_default.py``::
```

Comment:
We need to rephrase "conanfile profile" to something else, because we don't want to encourage calling these "profiles".

Maybe now is also a good time to remove "profile" from cloe-launch.

Reply:
Will we go forward with the cloe-launch change? I thought first you wanted to ask some stake holders to confirm if we do it or not.

Reply:
I have done the rephrase of the sentence.

Reply:
I would do the `cloe-launch` change at a separate time.